chaco/abstract_data_source.py

```diff
@@ -56,7 +56,7 @@ class AbstractDataSource(HasTraits):
     # Abstract methods
     #------------------------------------------------------------------------
 
-    def get_data(self):
+    def get_data(self, lod=None):
```

Comment:
Should update the docstring below to describe the new optional argument

Reply:
Updated.